core/src/i18n/en.yaml

```
@@ -117,7 +119,67 @@ discovery-and-network:
 functions:
   title: Functions
   variable:
-    tooltip: This variable comes from a Resource. To get the value, follow the link.
+    buttons:
+      add: Add environment variable
+      open:
+        custom: Custom Variable
+        secret: Secret Variable
+        config-map: Config Map Variable
```

```suggestion
        config-map: ConfigMap Variable
```

Official spelling, according to [Kubernetes docu](https://kubernetes.io/docs/concepts/configuration/configmap/).